apps/console/src/features/organization-discovery/components/edit-organization-discovery-domains.tsx

```diff
@@ -169,6 +172,25 @@ const EditOrganizationDiscoveryDomains: FunctionComponent<EditOrganizationDiscov
         return <ContentLoader dimmer />;
     }
 
+    /**
+     * Function to validate the input string is an email domain.
+     *
+     * @param values - Email domains.
+     */
+    const validateEmailDomain = (emailDomainList: string[]) => {
+
+        const emailDomainValidation: boolean = FormValidation.domain(emailDomainList[emailDomainList.length-1]);
+
+        if (!emailDomainValidation) {
+            setIsEmailDomainDataError(true);
+            setEmailDomainDataError( t(
```

Comment:
Seems like this is the only value being set to emailDomainDataError. WDYT of removing the state variable for error msg and directly using the error message in other places.

Reply:
Fixed with 59ca7063d0007513c62b470cde9b726f017412ee